diff -ur libopusenc-d53a41b5ed02eba854d12b6d35bea13a330018d5/include/opusenc.h libopusenc-src/include/opusenc.h
--- libopusenc-d53a41b5ed02eba854d12b6d35bea13a330018d5/include/opusenc.h	2023-05-03 05:48:47
+++ libopusenc-src/include/opusenc.h	2023-10-13 23:50:26
@@ -50,12 +50,12 @@
    <tt>libopus</tt>.
 */
 
+#include <stddef.h>
+#include <opus/opus.h>
+
 # if defined(__cplusplus)
 extern "C" {
 # endif
-
-#include <stddef.h>
-#include <opus.h>
 
 #ifndef OPE_EXPORT
 # if defined(WIN32)
