diff -ru opus-1.4/include/opus.h opus-src/include/opus.h
--- opus-1.4/include/opus.h	2023-01-30 11:13:11
+++ opus-src/include/opus.h	2023-10-13 23:12:09
@@ -33,8 +33,8 @@
 #ifndef OPUS_H
 #define OPUS_H
 
-#include "opus_types.h"
-#include "opus_defines.h"
+#include <opus/opus_types.h>
+#include <opus/opus_defines.h>
 
 #ifdef __cplusplus
 extern "C" {
diff -ru opus-1.4/include/opus_defines.h opus-src/include/opus_defines.h
--- opus-1.4/include/opus_defines.h	2023-01-30 11:13:11
+++ opus-src/include/opus_defines.h	2023-10-13 23:11:50
@@ -33,7 +33,7 @@
 #ifndef OPUS_DEFINES_H
 #define OPUS_DEFINES_H
 
-#include "opus_types.h"
+#include <opus/opus_types.h>
 
 #ifdef __cplusplus
 extern "C" {
diff -ru opus-1.4/include/opus_multistream.h opus-src/include/opus_multistream.h
--- opus-1.4/include/opus_multistream.h	2021-04-30 00:40:20
+++ opus-src/include/opus_multistream.h	2023-10-13 23:11:54
@@ -33,7 +33,7 @@
 #ifndef OPUS_MULTISTREAM_H
 #define OPUS_MULTISTREAM_H
 
-#include "opus.h"
+#include <opus/opus.h>
 
 #ifdef __cplusplus
 extern "C" {
diff -ru opus-1.4/include/opus_projection.h opus-src/include/opus_projection.h
--- opus-1.4/include/opus_projection.h	2021-04-30 00:40:20
+++ opus-src/include/opus_projection.h	2023-10-13 23:12:00
@@ -33,7 +33,7 @@
 #ifndef OPUS_PROJECTION_H
 #define OPUS_PROJECTION_H
 
-#include "opus_multistream.h"
+#include <opus/opus_multistream.h>
 
 #ifdef __cplusplus
 extern "C" {
