Only in flac-src: config.h
diff -ur flac-1.3.4/include/FLAC/all.h flac-src/include/FLAC/all.h
--- flac-1.3.4/include/FLAC/all.h	2022-02-20 11:46:15.000000000 -0600
+++ flac-src/include/FLAC/all.h	2022-07-16 22:11:35.000000000 -0500
@@ -33,15 +33,15 @@
 #ifndef FLAC__ALL_H
 #define FLAC__ALL_H
 
-#include "export.h"
+#include <FLAC/export.h>
 
-#include "assert.h"
-#include "callback.h"
-#include "format.h"
-#include "metadata.h"
-#include "ordinals.h"
-#include "stream_decoder.h"
-#include "stream_encoder.h"
+#include <FLAC/assert.h>
+#include <FLAC/callback.h>
+#include <FLAC/format.h>
+#include <FLAC/metadata.h>
+#include <FLAC/ordinals.h>
+#include <FLAC/stream_decoder.h>
+#include <FLAC/stream_encoder.h>
 
 /** \mainpage
  *
diff -ur flac-1.3.4/include/FLAC/callback.h flac-src/include/FLAC/callback.h
--- flac-1.3.4/include/FLAC/callback.h	2022-02-20 11:46:15.000000000 -0600
+++ flac-src/include/FLAC/callback.h	2022-07-16 22:11:25.000000000 -0500
@@ -33,7 +33,7 @@
 #ifndef FLAC__CALLBACK_H
 #define FLAC__CALLBACK_H
 
-#include "ordinals.h"
+#include <FLAC/ordinals.h>
 #include <stdlib.h> /* for size_t */
 
 /** \file include/FLAC/callback.h
diff -ur flac-1.3.4/include/FLAC/format.h flac-src/include/FLAC/format.h
--- flac-1.3.4/include/FLAC/format.h	2022-02-20 11:46:15.000000000 -0600
+++ flac-src/include/FLAC/format.h	2022-07-16 22:11:21.000000000 -0500
@@ -33,8 +33,8 @@
 #ifndef FLAC__FORMAT_H
 #define FLAC__FORMAT_H
 
-#include "export.h"
-#include "ordinals.h"
+#include <FLAC/export.h>
+#include <FLAC/ordinals.h>
 
 #ifdef __cplusplus
 extern "C" {
diff -ur flac-1.3.4/include/FLAC/metadata.h flac-src/include/FLAC/metadata.h
--- flac-1.3.4/include/FLAC/metadata.h	2022-02-20 11:46:15.000000000 -0600
+++ flac-src/include/FLAC/metadata.h	2022-07-16 22:11:14.000000000 -0500
@@ -34,9 +34,9 @@
 #define FLAC__METADATA_H
 
 #include <sys/types.h> /* for off_t */
-#include "export.h"
-#include "callback.h"
-#include "format.h"
+#include <FLAC/export.h>
+#include <FLAC/callback.h>
+#include <FLAC/format.h>
 
 /* --------------------------------------------------------------------
    (For an example of how all these routines are used, see the source
diff -ur flac-1.3.4/include/FLAC/stream_decoder.h flac-src/include/FLAC/stream_decoder.h
--- flac-1.3.4/include/FLAC/stream_decoder.h	2022-02-20 11:46:15.000000000 -0600
+++ flac-src/include/FLAC/stream_decoder.h	2022-07-16 22:11:48.000000000 -0500
@@ -34,8 +34,8 @@
 #define FLAC__STREAM_DECODER_H
 
 #include <stdio.h> /* for FILE */
-#include "export.h"
-#include "format.h"
+#include <FLAC/export.h>
+#include <FLAC/format.h>
 
 #ifdef __cplusplus
 extern "C" {
diff -ur flac-1.3.4/include/FLAC/stream_encoder.h flac-src/include/FLAC/stream_encoder.h
--- flac-1.3.4/include/FLAC/stream_encoder.h	2022-02-20 11:46:15.000000000 -0600
+++ flac-src/include/FLAC/stream_encoder.h	2022-07-16 22:12:14.000000000 -0500
@@ -34,9 +34,9 @@
 #define FLAC__STREAM_ENCODER_H
 
 #include <stdio.h> /* for FILE */
-#include "export.h"
-#include "format.h"
-#include "stream_decoder.h"
+#include <FLAC/export.h>
+#include <FLAC/format.h>
+#include <FLAC/stream_decoder.h>
 
 #ifdef __cplusplus
 extern "C" {
